### PR TITLE
Fail if there are not enough disks for testing

### PR DESCRIPTION
**Pull request by yontalcar**
Tests used to automatically pass when not enough unused disks were found.

### CODE REVIEW

File: tests/get_unused_disk.yml [hunk 1]
```diff
@@ -9,12 +9,10 @@
...
-- block:
-    - name: Exit playbook when there's no unused disks in the system
-      debug:
-        msg: "Unable to find unused disks. Exiting playbook."
-    - meta: end_play
-  when: unused_disks is undefined
+- name: Exit playbook when there's not enough unused disks in the system
+  fail:
+    msg: "Unable to find enough unused disks. Exiting playbook."
+  when: unused_disks is undefined or unused_disks|length < disks_needed|default(1)
```
Comment (pcahyna) [inline tests/get_unused_disk.yml]: `unused_disks is undefined`
This does not seem correct, this will fail almost all tests.

Comment (pcahyna) [reply tests/get_unused_disk.yml]: Sorry, I thought it was `disks_needed`

**Comment by tyll** [comment]
> looks good, but I will wait with merging until the more substantial PRs are merged.

To avoid too many PRs/changes to pile up it would be better to merge PRs faster.

**Comment by pcahyna** [comment]
> > looks good, but I will wait with merging until the more substantial PRs are merged.
> 
> To avoid too many PRs/changes to pile up it would be better to merge PRs faster.

Sure. I propose to merge PR #43 now

**Comment by pcahyna** [comment]
[citest bad]

**Pull request merged**